libs/hdf-converters/src/trufflehog-mapper.ts

```diff
@@ -0,0 +1,97 @@
+import {ExecJSON} from 'inspecjs';
+import _ from 'lodash';
+import {version as HeimdallToolsVersion} from '../package.json';
+import {BaseConverter, ILookupPath, MappedTransform} from './base-converter';
+
+export class TrufflehogMapper extends BaseConverter {
+  withRaw: boolean;
+
+  mappings: MappedTransform<
+    ExecJSON.Execution & {passthrough: unknown},
+    ILookupPath
+  > = {
+    platform: {
+      name: 'Heimdall Tools',
+      release: HeimdallToolsVersion
+    },
+    version: HeimdallToolsVersion,
+    statistics: {},
+    profiles: [
+      {
+        name: {path: 'wrapper[0].SourceName'},
+        title: {path: 'wrapper[0].SourceName'},
+        supports: [],
+        attributes: [],
+        groups: [],
+        status: 'loaded',
+        controls: [
+          {
+            path: 'wrapper',
+            key: 'id',
+            tags: {
+              nist: 'IA-5(7)',
```

Comment:
this should probably be an array of size one, not a string

Reply:
resolved to use array of size one